sdk/core/Azure.Core.TestFramework/src/TestLogger.cs

```diff
@@ -20,7 +20,7 @@ namespace Azure.Core.TestFramework
     /// We create an instance in SearchTestBase when we want to log test
     /// output.
     /// </summary>
-    internal class TestLogger : IDisposable
+    public class TestLogger : IDisposable
```

Comment:
Why do we need this to be public?

Reply:
I need this logger in the new ManagementRecordedTestBase.cs. I would like to move this into /sdk/core/Azure.Core.TestFramework/src/OneTimeSetupRecordBase.cs. Problem is this class would then need access to ResourceGroupCleanupPolicy which currently is in /common/ManagementTestShared/ResourceGroupCleanupPolicy.cs

That class should be moved as well to sdk/core/ (as you have said before). This would make the PR larger, but I'd be willing to make that change now.